feeds/pl.json

```
@@ -188,6 +205,16 @@
             "type": "mobility-database",
             "mdb-id": 2086,
             "fix": true
+        },
+        {
+            "name": "Koleje-Śląskie",
+            "type": "transitland-atlas",
+            "transitland-atlas-id": "f-u2v-koleje~slaskie"
```

```suggestion
            "transitland-atlas-id": "f-u2v-koleje~slaskie",
            "url-override": "https://koleje-ks.pl/gtfs/2024-2025.zip"
```
(Or fix it upstream in transitland-atlas, the url seems to change every year)